.utils/terraform_run_all.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+
+
+#
+# bash .utils/terraform_run_all.sh <Action>
+# bash .utils/terraform_run_all.sh init
+#
+
+# 'set -e' tells the shell to exit if any of the foreground command fails,
+# i.e. exits with a non-zero status.
+set -eu
+
+pids=()
+ACTION="$1"
+
+policy_folders=(
+  'src/01_custom_roles'
+  'src/02_policy_audit_logs'
+  'src/02_policy_resource_lock'
+  'src/02_policy_tags_inherit_from_subscription'
+  'src/03_policy_set'
+  'src/04_policy_assignments'
+)
+
+function rm_terraform {
```

Comment:
ma questa cosa è terribilmente inefficiente quando lavori in locale.
e nelle pipeline non serve

Reply:
ok allora la tolgo